docs/releases/v13/version-v13.0.0.md

```diff
@@ -92,7 +92,27 @@ When a record is shared, its inclusion requests will be also accessible. There i
 
 To be completed
 
-### Collections
+### Communities
+
+#### Themed communities
+
+Communities can now have their own theming with a custom font and colors, which apply to all community pages including records and requests. Below is an example of two themed communities on Zenodo.
+
+![A default community and two themed communities on Zenodo](imgs/themed-communities.png)
+
+Themed communities benefit from a custom homepage, defined via HTML template in `<instance>/templates/themes/<theme>/invenio_communities/details/home/index.html`.
+
+#### Subcommunities
+
+It is now possible to create heirarchical relationships between communities, allowing for departments, subject areas and other structures to be represented via related communities. Records from the "child" community are automatically indexed in the "parent" community, allowing all the records of the children to be browsed in the parents. The communities are also bidirectionally linked so that it is easy to navigate between both.
```

Comment:
```suggestion
It is now possible to create hierarchical relationships between communities, allowing for departments, subject areas and other structures to be represented via related communities. Records from the "child" community are automatically indexed in the "parent" community, allowing all the records of the children to be browsed in the parents. The communities are also bidirectionally linked so that it is easy to navigate between both.
```